release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -74,6 +74,10 @@ To use IBM Secure Execution, you must have host keys for your host machine(s) an
 
 For more information, see xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-rhcos-using-ibm-secure-execution_installing-ibm-z-kvm[Installing {op-system} using IBM Secure Execution].
 
+[id="ocp-4-13-lsof-rhcos"]
+==== {op-system} now includes lsof
+{product-title} {product-version} now includes the `lsof` command by default in {op-system}.
```

Comment:
If there's no particular reason "by default" was added, I think "now includes the `lsof` command in {op-system}" is sufficient.